templates/company/test/company.js

```diff
@@ -1,8 +1,9 @@
 const { hash: namehash } = require('eth-ens-namehash')
```

Comment:
Whereever we add gas tests, it would be nice to also include one for `newTokenAndInstance()` (and the like that combine multiple steps into one).

Reply:
sure, will do